pom.xml

```diff
@@ -1,151 +1,128 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>io.quarkus</groupId>
-  <artifactId>quarkus-spring-boot-api</artifactId>
-  <version>2.2.Alpha1-SNAPSHOT</version>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-spring-boot-api</artifactId>
+    <version>2.2.Alpha1-SNAPSHOT</version>
 
-  <name>Spring Boot dependencies for Quarkus Spring Spring Boot Properties extension</name>
-  <description>The minimum dependencies to reduce the footprint of Quarkus applications using the Spring Spring Boot Properties extension</description>
-  <url>https://quarkus.io/</url>
-  <developers>
-    <developer>
-      <id>jboss.org</id>
-      <name>JBoss.org Community</name>
-      <organization>JBoss.org</organization>
-      <organizationUrl>http://www.jboss.org</organizationUrl>
-    </developer>
-  </developers>
+    <name>Spring Boot dependencies for Quarkus Spring Spring Boot Properties extension</name>
+    <description>The minimum dependencies to reduce the footprint of Quarkus applications using the Spring Spring Boot
```

Comment:
This would have to change the proper description added to each module

Reply:
:face_palm:

Reply:
:woman_facepalming: